frontend/manager/Attachments.vue

```diff
@@ -23,6 +23,9 @@ const props = defineProps({
     permission: String
 });
 
+const emit = defineEmits(['update:attachmentCount','update:isAttachmentsEditable']); // Emitting the number of attachments to parent component
```

Comment:
Why do you need signals? Please avoid signals, use reactive variables or models.